packages/gatsby/cache-dir/page-renderer.js

```diff
@@ -16,9 +17,13 @@ class PageRenderer extends React.Component {
       loader: publicLoader,
     })
 
+    const AsyncPageComponent = loadable(() =>
+      this.props.pageResources.component()
```

Comment:
you are sure it's a function and not the value itself? in line 26 it was before not a function
```suggestion
      this.props.pageResources.component
```

Reply:
@muescha I believe it became a function due to another change in this PR. @wardpeet and I were working on this together and we were surprised that it was a function as well.

Reply:
so then it's ok